panther_analysis_tool/main.py

```diff
@@ -1007,6 +1018,10 @@ def run() -> None:
     if getattr(args, "filter", None) is not None:
         args.filter = parse_filter(args.filter)
 
+    if bool(getattr(args, "ignore_extra_keys", None)):
+        RULE_SCHEMA._ignore_extra_keys = True  # pylint: disable=protected-access
+        POLICY_SCHEMA._ignore_extra_keys = True  # pylint: disable=protected-access
```

Comment:
A brief comment explaining why we went this route (basically that the alternative is super ugly) will help future maintainers of this code I think.